addon/templates/components/frost-list-item-content.hbs

```diff
@@ -0,0 +1,67 @@
+<div class={{concat
+    'frost-list-item-container'
+    (if (eq index 0) ' first')
+    (if (eq index (sub _items.length 1)) ' last')
+    (if model.states.isSelected ' is-selected')
+  }}
+  data-test={{hook (concat hook '-item-container') index=index }}
+>
+  <div class='frost-list-item-container-base'>
+    {{#if (and (or itemExpansion isTypedItemWithExpansion) (not isTypedItemWithoutExpansion))}}
+      {{frost-list-item-expansion
+        hook=(concat hookPrefix '-expansion')
+        hookQualifiers=(hash index=index)
+        model=model.content
+        isExpanded=model.states.isExpanded
+        onExpand=onExpand
+      }}
+    {{else if isAnyTypedItemExpansion}}
+      <div class="item-expansion-placeholder"></div>
```

Comment:
Why do we need this?

Reply:
Yes this is for the alignment when there are items with expansion and items without expansion, since the expansion icon/button does not show up when the item does not have expansion.